workflow/usage.rst

```diff
@@ -450,3 +459,220 @@ The following example shows these functions in action:
+
+                    <framework:transition name="to_review">
+                        <framework:from>draft</framework:from>
+                        <framework:to>review</framework:to>
+                        <framework:metadata>
+                            <framework:priority>0.5</framework:priority>
+                        </framework:metadata>
+                    </framework:transition>
+                    <!-- ... -->
+                </framework:workflow>
+            </framework:config>
+        </container>
+
+    .. code-block:: php
+
+        // config/packages/workflow.php
+
+$container->loadFromExtension('framework', [
+    // ...
+    'workflows' => [
+        'blog_publishing' => [
+            'metadata' => [
+                'title' => 'Blog Publishing Workflow',
+            ],
+            // ...
+            'places' => [
+                'draft' => [
+                    'metadata' => [
+                        'max_num_of_words' => 500,
+                    ],
+                ],
+                // ...
+            ],
+            'transitions' => [
+                'to_review' => [
+                    'from' => 'draft',
+                    'to' => 'review',
+                    'metadata' => [
+                        'priority' => 0.5,
+                    ],
+                ],
+            ],
+        ],
+    ],
+]);
+
+Then you can access this metadata in your controller as follows::
+
+    public function myControllerAction(Registry $registry, Article $article)
+    {
+        $workflow = $registry->get($article);
+
+        $workflow
+            ->getMetadataStore()
+            ->getWorkflowMetadata()['title'] ?? false
+        ;
+
+        // or
+        $workflow
+            ->getMetadataStore()
+            ->getPlaceMetadata('draft')['title'] ?? false
+        ;
+
+        // or
+        $aTransition = $workflow->getDefinition()->getTransitions()[0];
+        $workflow
+            ->getMetadataStore()
+            ->getTransitionMetadata($aTransition)['title'] ?? false
+        ;
+    }
+
+There is a shortcut that works with everything::
+
+    $workflow
+        ->getMetadataStore()
+        ->getMetadata('title')
+    ;
+
+In a Flash message in your Controller::
+
+            // $transition = ...; (an instance of Transition)
+            // $workflow is a Workflow instance retrieved from the Registry (see above)
+            $title = $workflow->getMetadataStore()->getMetadata('title', $transition);
+            $this->addFlash('info', "You have successfully applied the transition with title: '$title'");
+
+Metadata can also be accessed in a Listener, from the Event object.
+
+Using transition blockers you can
+return a user-friendly error message when you stop a transition from happening. In the example we
+get this message from the :class:`Symfony\\Component\\Workflow\\Event\\Event`'s metadata, giving
+you an easy place to manage the text. This is a contrived example; in production code you may
```

Comment:
I think we can remove the entire phrase that starts with *"This is a contrived example..."* and add instead a comment like this one just before the `$explanation = ...` code:

```
// inject the 'translation' service if you need to translate the explanation
```

Reply:
I've simplified the english and turned it into a tip, as it's an aisde rather than part of the narrative.